x-pack/plugins/alerting/server/task_runner/rule_loader.ts

```diff
@@ -134,6 +135,9 @@ export async function getRuleAttributes<Params extends RuleTypeParams>(
       { namespace }
     );
   } catch (e) {
+    if (isBoom(e, 404)) {
```

Comment:
We should use SavedObjectsErrorHelpers.isNotFoundError(e) provided by `@kbn/core/server`

Reply:
Done.